ticketutil/jira.py

```diff
@@ -21,9 +21,19 @@ def __init__(self, url, project, auth=None, ticket_id=None, verify=False):
         self.url = url[:-1] if url.endswith('/') else url
         self.rest_url = '{0}/rest/api/2/issue'.format(self.url)
         self.ticket_content = None
+        # HTTP Basic Auth
         if isinstance(auth, tuple):
             self.auth = auth
             self.auth_url = self.url
+        # Personal Access Token Auth
+        if isinstance(auth, dict):
```

Comment:
```suggestion
        elif isinstance(auth, dict):
```

Reply:
Need this change for basic auth to continue to work, otherwise we traceback on auth project key verification.

Example:
Project PROJECTKEY is not valid
Traceback (most recent call last):
  File "/home/crag/Documents/git/ticketutil/./jira_test.py", line 17, in <module>
    t = JiraTicket('https://jira.example.com', 'PROJECTKEY',
  File "/home/crag/Documents/git/ticketutil/ticketutil/jira.py", line 42, in __init__
    super(JiraTicket, self).__init__(project, ticket_id, verify=verify)
  File "/home/crag/Documents/git/ticketutil/ticketutil/ticket.py", line 41, in __init__
    raise TicketException("Project {0} is not valid".format(self.project))
ticketutil.ticket.TicketException: Project PROJECTKEY is not valid

Reply:
Need this change for basic auth to continue to work, otherwise we traceback on auth project key verification.

Example:
Project PROJECTKEY is not valid
Traceback (most recent call last):
File "/home/crag/Documents/git/ticketutil/./jira_test.py", line 17, in
t = JiraTicket('https://jira.example.com', 'PROJECTKEY',
File "/home/crag/Documents/git/ticketutil/ticketutil/jira.py", line 42, in init
super(JiraTicket, self).init(project, ticket_id, verify=verify)
File "/home/crag/Documents/git/ticketutil/ticketutil/ticket.py", line 41, in init
raise TicketException("Project {0} is not valid".format(self.project))
ticketutil.ticket.TicketException: Project PROJECTKEY is not valid

Reply:
Thanks @hluk and @Aftermath ! Fixing this now.